src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = people.filter((person) => person.sex === 'm'
+    && ((!century)
+      ? true
+      : Math.ceil(person.died / 100) === century)
+  );
+  const averageAge = (man
+    .reduce((acc, person) => acc + (person.died - person.born), 0)
+  ) / man.length;
```

Comment:
```suggestion
  const averageAge = (man
    .reduce((acc, person) => 
      acc + (person.died - person.born), 0)) / man.length;
```

Don't translate arithmetical operators on new lines